components/BlogListing.js

```diff
@@ -0,0 +1,67 @@
+const dotCMSApi = require('../utils/dotcms/dotcmsApi');
+
+import Link from 'next/link';
+import DotcmsImage from './DotcmsImage';
+import Loader from './Loader';
+import useDotCMSApi from '../hooks/useDotCMSApi';
+
+const DateText = ({ value }) => {
```

Comment:
move this to the `utilities` file

Reply:
I'm gonna live inside `Components` because is a component I think this components might be replaced with something else.

Reply:
Replace the one we had with this one and now lives in `Shared`